.github/workflows/pull-request.yml

```diff
@@ -5,10 +5,15 @@ on:
     branches: '*'
 
 jobs:
+  fossa-scans:
+    name: Rune FOSSA Scans
```

Comment:
Is the extra `e` a typo?

Reply:
![image](https://github.com/tidal-music/tidal-sdk-web/assets/25121161/bdf660e3-89a5-4433-8f32-b1aeb0f19a9a)

Reply:
Rune was here! :-P